components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -138,6 +139,11 @@ void ViewCounterService::BrandedWallpaperWillBeDisplayed(
     const std::string& campaign_id,
     const std::string& creative_instance_id,
     bool should_metrics_fallback_to_p3a) {
+  if (!brave_ads::kShouldSupportNewTabPageAdConfirmationsForNonRewards.Get()) {
```

Comment:
Was `should_metrics_fallback_to_p3a` already set to true here: [ntp_sponsored_images_data.cc](https://github.com/brave/brave-core/pull/28848/files#diff-f842855d1555cc187a2162dda9b33ae36e4e8bbff193d3e36c5d8af64533ec3a)?

Reply:
Probably yes, but we’re adding guards for safety for now.